sdk/ml/azure-ai-ml/azure/ai/ml/_schema/job/services.py

```diff
@@ -25,13 +44,70 @@ class JobServiceSchema(PathAwareSchema):
             fields.Str(),
         ]
     )
-    port = fields.Int()
-    endpoint = fields.Str(dump_only=True)
-    status = fields.Str(dump_only=True)
-    nodes = fields.Str()
-    error_message = fields.Str(dump_only=True)
-    properties = fields.Dict()
 
     @post_load
     def make(self, data, **kwargs):  # pylint: disable=unused-argument,no-self-use
         return JobService(**data)
+
+
+class TensorBoardJobServiceSchema(JobServiceBaseSchema):
+    job_service_type = UnionField(
```

Comment:
UnionField is unnecessary if there's only one possible schema.

Reply:
done